include/Lv2Ports.h

```diff
@@ -184,15 +185,15 @@ struct Audio : public VisitablePort<Audio, PortBase>
 
 	//! Copy buffer passed by LMMS into our ports
 	//! @param channel channel index into each sample frame
-	void copyBuffersFromCore(const sampleFrame *lmmsBuf,
+	void copyBuffersFromCore(const SampleFrame *lmmsBuf,
```

Comment:
```suggestion
	void copyBuffersFromCore(const SampleFrame* lmmsBuf,
```

Felt like specifying this.

Reply:
We can leave this out too.